extensions/ql-vscode/src/vscode-tests/cli-integration/run-cli.test.ts

```diff
@@ -60,4 +61,11 @@ describe('Use cli', function() {
       expect(languages).to.have.property(expectedLanguage).that.is.not.undefined;
     }
   });
+
+  it('should resolve query by language', async function() {
```

Comment:
Sometimes it is useful to test negative results. If you wish, you can add a second test query that looks like the original (no import statement), and a corresponding test function that checks `queryInfo.byLanguage` is an empty object.

Reply:
Hmm, I've been trying to test a non-language query, but it keeps resolving the language as `javascript` anyway 🤔 Maybe because the CLI picks up on a nearby qlpack.yml file? I'll try putting a test query in some different locations!

Reply:
(I haven't found a nice way to do this, so I'm going to skip this for now...)